yarn.lock

```diff
@@ -112,6 +112,10 @@
   version "0.0.38"
   resolved "https://registry.yarnpkg.com/@types/estree/-/estree-0.0.38.tgz#c1be40aa933723c608820a99a373a16d215a1ca2"
 
+"@types/node@^10.5.5":
```

Comment:
Unsure why yarn decided to update these, I can force reset if it's a problem

Reply:
because someone pushed a change to package.json without checking the updated yarn.lock

Reply:
Mind making an issue so we can ensure CI fails in that scenario (when package.json is out of sync with yarn.lock)?

Reply:
Done! #16920

Reply:
That got fixed now, so we should drop the changes here to yarn.lock...